scripts/build_places.exs

```diff
@@ -598,9 +614,14 @@ pa_ess_screens =
 
 # Merge screens and pa/ess
 merged_paess =
-  Enum.map(contents, fn %{id: id, screens: screens} = place ->
+  contents
+  |> Enum.map(fn %{id: id, screens: screens} = place ->
     Map.put(place, :screens, screens ++ (pa_ess_screens[id] || []))
   end)
+  # Get rid of CR and Bus stops with no screens
+  |> Enum.reject(fn %{routes: routes, screens: screens} ->
+    cr_or_bus_only?.(routes) and length(screens) == 0
```

Comment:
**quibble (non-blocking):** i find `Enum.empty?/1` a bit easier to follow. bonus! it will be slightly faster because it will check for `value == []` instead of iterating over the entire length of the list.

```suggestion
    cr_or_bus_only?.(routes) and Enum.empty?(screens)
```